README.md

```
@@ -71,19 +71,17 @@ docker run --rm -p 8080:8080 \
 
-ℹ️ You can configure GoModel with `.env` variables, a `config.yaml` file, OR directly in the dashboard.
+ℹ️ Configure GoModel with `.env`, a `config.yaml` file, or manage the most important settings directly in the dashboard.
 
-ℹ️ Full list of environment variables (including all available providers): [`.env.template`](./.env.template)
-
-ℹ️ The most secure way in production is to use `.env` to load API keys.
+ℹ️ See [`.env.template`](./.env.template) for the complete list of environment variables, including all available providers.
 
 **Step 2:** Open the dashboard
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)